docs/content/en/contribute/software/prerequisite/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Software that is required
+description: Information about installing the tools before using Keptn Lifecycle Toolkit
+weight: 200
+---
+
+
+## Prerequisites
+
```

Comment:
What would you think about a little intro sentence here?    Something like, "The following software is required to set up your Keptn Lifecycle Controller development environment.  Or steal the prose that follows the list.

It might also be good to comment that most of these are not required if you only want to contribute to the documentation and do not need to run KLT software as research for the doc contributions you are doing.

Some people disagree but I think that most lists should be preceded by a statement about what the list is.

Reply:
your suggestions always make sense to me👻I will add this information 😉

Reply:
I already added this thing but it's at bottom :(